src/server/game/Achievements/AchievementMgr.cpp

```diff
@@ -65,7 +65,7 @@ bool AchievementMgr::CanUpdateCriteriaTree(Criteria const* criteria, CriteriaTre
 {
     AchievementEntry const* achievement = tree->Achievement;
     if (!achievement)
-        return false;
+        return true;
```

Comment:
This correction needs an additional check? 6688ad03d35bb264e41f5fbfcc5cee377c84d913